inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -181,6 +179,48 @@ std::vector<std::shared_ptr<ngraph::op::util::SubGraphOp::OutputDescription>> V1
     return outputs;
 }
 
+ngraph::op::v5::Loop::SpecialBodyPorts V10Parser::XmlDeserializer::parsePurposeAttribute(const pugi::xml_node& node) {
+    ngraph::op::v5::Loop::SpecialBodyPorts result = {-1, -1};
+    std::map<uint64_t, uint64_t> params;
+    std::map<uint64_t, uint64_t> results;
+    map_type_in_function(node, "Parameter", params);
```

Comment:
IMHO, this function should return `std::map` as value.

Reply:
Corrected.